docs/config/cluster.rst

```diff
@@ -714,26 +714,31 @@ Allow to control the allocation of all shards based on include/exclude filters.
 E.g. this could be used to allocate all the new shards on the nodes with
 specific IP addresses or custom attributes.
 
+.. NOTE::
+
+  If you apply allocation filters, non-matching nodes are deallocated after
```

Comment:
`non-matching nodes are deallocated` might be interpreted as that the nodes are stopped or something like that, which is not the case.

Maybe re-work the paragraph above as well to incorporate this. E.g.


> Allows to control whether shards can be allocated onto a node. If a setting prevents shards from being allocated to a node this means that both, new shards won't be allocated onto those nodes, and that existing shards which were on the node before but no longer fit onto the node based on the updated criteria will be moved away.

Reply:
I would drop the "Allows to control whether shards can be allocated onto a node." sentence from the NOTE block. but otherwise, this rewording looks good

Reply:
@mfussenegger i understood your comment as request to drop the `Note` entirely (which i did) i put it there similar as you would have phrased it.

@mechanomi i guess you want to rephrase it 😬